extensions/wikia/UserTools/templates/UserTools_Toolbar.php

```diff
@@ -1,4 +1,4 @@
-<?= F::app()->renderView( 'UserTools', 'Menu', array( 'format' => 'html', 'items' => $toolbar )); ?>
+<?= F::app()->renderView( 'UserTools', 'Menu', [ 'format' => 'html', 'type' => 'main', 'items' => $toolbar ] ); ?>
```

Comment:
A little bit of docs saying what are the possible types are (submenu/main) would be nice. :)